### ISSUE TITLE
Remote trauma history

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10718]
**Preferred term label:**
Remote trauma history

**Synonyms**

**Definition (free text, please give PubMed ID)**
history of trauma months to years ago, including but not limited to fracture or other orthopedic injury at the symptomatic area

**Parent term (use hpo.jax.org/app)**
We used HP:0031135 for our finding of "Trauma, recent history", but the orthopedists find it very useful to distinguish recent from remote history. 

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Scaphoid nonunion advanced collapse (SNAC)
Scapholunate advanced collapse (SLAC)
Distal radioulnar joint (DRUJ) arthritis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001129
- label: Remote trauma history
- parents: [Exposure history (HP:4000110)](https://hpo.jax.org/app/browse/term/HP:4000110)
- pmid: 
- def: History of trauma months to years ago, including but not limited to fracture or other orthopedic injury at the symptomatic area.
- comment: This feature may suggest scaphoid nonunion advanced collapse, scapholunate advanced collapse, or distal radioulnar joint arthritis.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10718
- synonyms:
